app/assets/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,47 @@
+.header-nav {
+	align-content: center;
+	background-color: $header_color;
+	padding-left: 1rem;
+
+	.bar{
+		display: flex;
+		font-size: 2rem;
+		gap: 1rem;
+		padding: 1.5rem;
+
+		.tittle{
+			font-size: 3rem;
+		}
+
+		.logout{
+			font-size: 2.5rem;
+			margin-left: auto;
+		}
+
+		a{
+			color: $color_white;
+			text-decoration: none;
+		}
+
+		.input_search{
+			background-color: $background_color;
+			margin-left: 3rem;
+			width: 50%;
+		}
+
+		select {
+			@extend .input_search;
+			width: 9rem;
+		}
+
+		.button_search {
+			@extend .input_search;
+			background-color: $background_color;
+			background-position: center center;
+			background-repeat: no-repeat;
+			background-size: 3.5rem;
+			background: url(search.svg);
+			width: 7rem;
+		}
+	}
+}
```

Comment:
Check the indentation of your file.

Reply:
@YilKanda , i think it is a problem with github, logically the indentation in this file works well, can test it review the spaces with your cursor, another posibility is that they can have in vs code settings the tab sizes in 4 spaces even in this block with indentation in 2 spaces git takes the another configutarion

Reply:
I don't understand why does it look like this here, in the file we have 2 spaces for identation and we don't know how to solve this

Reply:
Solved